src/HealthcareApis/HealthcareApis/Common/HealthcareApisArgumentValidator.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.Azure.Commands.HealthcareApis.Properties;
+using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.Azure.PowerShell.Cmdlets.HealthcareApis.Common
+{
+    public static class HealthcareApisArgumentValidator
+    {
+
+        public static bool ValidateObjectId(string AccessPolicyObjectId)
```

Comment:
@poadhika nit: please use camel case for variables: `AccessPolicyObjectId` --> `accessPolicyObjectId`